### DISCUSSION TITLE
Insert current Date and/or Time in the note

### DISCUSSION THREAD

**Discussion by giuseppepagnoni** [TriliumNext/Trilium#3487] [Q&A] [answered]
Hi, is there any key combination or {} to automatically add the current date or time to the text of a note?
Thanks!
G

**Comment by zadam** [comment, marked as answer]
Yes, `ALT-T` by default, can be changed in options:

![image](https://user-images.githubusercontent.com/617641/210635148-2d1fdcce-8018-4453-9ffb-f39b49c01297.png)

**Reply by giuseppepagnoni** [reply]
Thanks!

**Comment by Lipown** [comment]
Can only date be inserted? How?

**Reply by SiriusXT** [reply]
> Can only date be inserted? How?

You can insert the date and time in any format, and configure it in the options.

**Reply by Lipown** [reply]
I do want to insert date only without time - that does not seems to be configurable

**Reply by SiriusXT** [reply]
> I do want to insert date only without time - that does not seems to be configurable

What version of Trilium are you using? Get the latest version here: https://github.com/TriliumNext/Trilium/releases

**Reply by Lipown** [reply]
Docker with tag: latest 
<img width="539" height="289" alt="image" src="https://github.com/user-attachments/assets/99931dd6-59c4-42c1-81b6-679a075962f8" />

**Reply by SiriusXT** [reply]
Configuration in the options
![mmexport1762266772705](https://github.com/user-attachments/assets/00f5bbb6-6444-47aa-bd6d-773015b3cb51)

**Reply by Lipown** [reply]
Apologize my blindness sir. And thank you for pointing that out where it specifically is. I was looking alays to Language & Region. I am new with Trilium (came from Obsidian). I guess I will have more questions like this :)
